app/views/thread_mailer/_new_comment.html.haml

```diff
@@ -10,3 +10,5 @@
         = render_email_markdown comment.body
       - else
         = render_email_plaintext comment.body
+      - if comment.attachments.any?
+        = render "thread_mailer/attachment", comment: comment
```

Comment:
Minor naming nitpick: can we call this 'attachments', since this is rendering out multiple attachments within the partial?